docusaurus/docs/reactnative/guides/channel_background_customization.mdx

```diff
@@ -0,0 +1,318 @@
+---
+id: channel-background-customization
+sidebar_position: 13
+title: Channel Background Customization
+---
+
+You can implement this customization in your existing app or create a new one by running the following
+
+```shell
+npx react-native init MyApp --template stream-chat-react-native-template
```

Comment:
This template is not actively maintained by us. So I am a bit sceptical about recommending this. In my view, we can avoid this section. People would normally follow installing steps first fo through the tutorial and come to this guide when they want some customisation.

Reply:
Agree on this one!!

Reply:
Someone reading this section already has an app running on his machine ... either his own app or from tutorial

Reply:
Welllll, it's maintained by me 😅